### PR TITLE
Add Arduino curriculum

### PR DESCRIPTION
**Pull request by noabenefraim**
PR for Arduino curriculum

### CODE REVIEW

File: content/english/arduino/1-introduction-building-with-ardunio/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Introduction - Building with Arduino"
+description: ""
+draft: true
```
Comment (Anvesha4) [inline content/english/arduino/1-introduction-building-with-ardunio/_index.md:4]: Can you add prereq and difficulty here as well? just like other workshops?

File: content/english/arduino/3-settning-up-the-PC/_index.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+![Cannot load image](content\english\arduino\3-settning-up-the-PC\img2.png)
+
+Next, you must tell the Arduino which port you are using on your computer.
+
+To select the port, go to Tools then Port then select the port that says Arduino.
+
+![Cannot load image](content\english\arduino\3-settning-up-the-PC\img3.png)
+
+### Activity 1 - Blinking LED
```
Comment (Anvesha4) [inline content/english/arduino/3-settning-up-the-PC/_index.md:46]: We generally have separate activity pages with activities in different folders. Look at other workshops for reference

File: content/english/arduino/4-appendix/_index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+weight: 1
+translationKey: "arduino-4"
+---
+
+# Appendix
+
+## The components, in detail
+
+Imagine our circuit is a play. Before the final performance in front of an audience, we need a place to practice. This is where the breadboard comes in. 
+> > Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together.
```
Comment (ozhang22) [inline content/english/arduino/4-appendix/_index.md]: I'm not a huge fan of using the quote indent for this. Can we just use a bulleted list instead?

File: content/english/arduino/3-settning-up-the-PC/_index.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
+---
+title: "Setting up the PC"
```
Comment (ozhang22) [inline content/english/arduino/3-settning-up-the-PC/_index.md:2]: There's a typo in the file path for this file. Should be: 3-setting-up-the-PC/_index.md

File: content/english/arduino/3-settning-up-the-PC/_index.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+description: ""
+draft: true
+weight: 1
+translationKey: "arduino-3"
+---
+
+# Setting up the PC
+
+Download and install teh Arduino IDE from:
```
Comment (ozhang22) [inline content/english/arduino/3-settning-up-the-PC/_index.md:12]:
```suggestion
Download and install the Arduino IDE from:

```

File: content/english/arduino/3-settning-up-the-PC/_index.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+![Cannot load image](content\english\arduino\3-settning-up-the-PC\img2.png)
+
+Next, you must tell the Arduino which port you are using on your computer.
+
+To select the port, go to Tools then Port then select the port that says Arduino.
+
+![Cannot load image](content\english\arduino\3-settning-up-the-PC\img3.png)
+
+
+
```
Comment (ozhang22) [inline content/english/arduino/3-settning-up-the-PC/_index.md:48]: nit: remove extra lines at the end

File: content/english/arduino/3-settning-up-the-PC/_index.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+Once the board is connected, you will need to go to Tools then Board then finally select Arduino Uno.
+
+![Cannot load image](content\english\arduino\3-settning-up-the-PC\img2.png)
+
+Next, you must tell the Arduino which port you are using on your computer.
+
+To select the port, go to Tools then Port then select the port that says Arduino.
+
+![Cannot load image](content\english\arduino\3-settning-up-the-PC\img3.png)
```
Comment (ozhang22) [inline content/english/arduino/3-settning-up-the-PC/_index.md:45]: Fix this file path too
```suggestion
![Cannot load image](content\english\arduino\3-setting-up-the-PC\img3.png)

```

File: content/english/arduino/8-appendix/_index.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+weight: 1
+translationKey: "arduino-8"
+---
+
+# Appendix
+
+## The components, in detail
+
+Imagine our circuit is a play. Before the final performance in front of an audience, we need a place to practice. This is where the breadboard comes in. 
+> > Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together.
```
Comment (ozhang22) [inline content/english/arduino/8-appendix/_index.md:14]: I'm not a huge fan of using these quote indents. Can we either use a bulleted list, or maybe wrap in a notice block?

```suggestion
> > Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 

```
```suggestion
{{% notice tip % }}
Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 

...

{{% / notice % }}
```

File: content/english/arduino/1-introduction-building-with-ardunio/_index.md (hunk 1 above)
Comment (noabenefraim) [reply content/english/arduino/1-introduction-building-with-ardunio/_index.md:4]: Fixed

File: content/english/arduino/3-settning-up-the-PC/_index.md (hunk 4 above)
Comment (noabenefraim) [reply content/english/arduino/3-settning-up-the-PC/_index.md:2]: Resolved

File: content/english/arduino/3-settning-up-the-PC/_index.md (hunk 5 above)
Comment (noabenefraim) [reply content/english/arduino/3-settning-up-the-PC/_index.md:12]: Resolved

File: content/english/arduino/3-settning-up-the-PC/_index.md (hunk 6 above)
Comment (noabenefraim) [reply content/english/arduino/3-settning-up-the-PC/_index.md:48]: Resolved

File: content/english/arduino/3-settning-up-the-PC/_index.md (hunk 7 above)
Comment (noabenefraim) [reply content/english/arduino/3-settning-up-the-PC/_index.md:45]: Resolved

File: content/english/arduino/8-appendix/_index.md (hunk 8 above)
Comment (noabenefraim) [reply content/english/arduino/8-appendix/_index.md:14]: Fixed with bullet lists

**Pull request merged**